client/plots/summary.js

```diff
@@ -407,11 +407,21 @@ export async function getPlotConfig(opts, app) {
 				} else if (config.term?.q?.mode == 'continuous' || config.term2?.q?.mode == 'continuous') {
 					/** Fix for summary tabs switching to different plots when opening and closing
```

Comment:
please clarify purpose of this comment

Reply:
This comment has been removed in my commit.